ghost/ghost/src/db/knex/actor.repository.knex.ts

```diff
@@ -0,0 +1,148 @@
+import {Inject} from '@nestjs/common';
+import ObjectID from 'bson-objectid';
+import {ActorRepository} from '../../core/activitypub/actor.repository';
+import {Actor} from '../../core/activitypub/actor.entity';
+import {SettingsCache} from '../../common/types/settings-cache.type';
+
+interface DomainEvents {
+    dispatch(event: unknown): void
+}
+
+export class ActorRepositoryKnex implements ActorRepository {
+    private readonly domainEvents: DomainEvents;
+
+    #defaultActor: Actor;
+
+    constructor(
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        @Inject('knex') private readonly knex: any,
+        @Inject('SettingsCache') settingsCache: SettingsCache,
+        @Inject('DomainEvents') domainEvents: DomainEvents
+    ) {
+        this.#defaultActor = Actor.create({
```

Comment:
Same as above, but I was able to `create` a new actor here. Again, not sure if that is the right approach.

Reply:
Also not sure, if we need a default actor, or if this is really the right place to create them?

Reply:
Fixtures is the correct place for defaults, but here is good for now, we can easily make changes to the values in dev

Reply:
Although - this is going to blow away the following list each time? I think need to be careful about that